### PR TITLE
Add blog post for "the year in review (2023)"

### PR DESCRIPTION
**Pull request by odow**
Andrea may post this on the ORSNZ site (http://orsnz.org.nz), but it's useful here too.

Mostly a text dump at this point, I haven't really edited it.

### CODE REVIEW

File: _posts/2024-01-30-jump_year_in_review.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
+---
+layout: page
+title:  "JuMP: the year in review (2023)"
+date:   2024-01-30
+---
+
+_Author: Oscar Dowson_
```
Comment (mlubin) [inline _posts/2024-01-30-jump_year_in_review.md:7]: There's probably a metadata field for the author.

**Review by mlubin — APPROVED** [review]
LGTM overall. Might want another set of eyes to proofread.

File: _posts/2024-01-30-jump_year_in_review.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+like this:
+```julia
+julia> using JuMP, PATHSolver
+
+julia> function solve_insurance_problem(; pi = 0.01, L = 0.5, γ = 0.02, ρ = -1)
+           U(C) = -1 / C
+           MU(C) = 1 / C^2
+           model = Model(PATHSolver.Optimizer)
+           set_silent(model)
+           @variable(model, EU, start = 1)   # Expected utilitiy
```
Comment (chriscoey) [inline _posts/2024-01-30-jump_year_in_review.md:112]:
```suggestion
           @variable(model, EU, start = 1)   # Expected utility
```

File: _posts/2024-01-30-jump_year_in_review.md (hunk 1 above)
Comment (odow) [reply _posts/2024-01-30-jump_year_in_review.md:7]: There isn't. We could add a new layout, but maybe that's a separate PR.

File: _posts/2024-01-30-jump_year_in_review.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+2-element Vector{AffExpr}:
+ x[1] + 1
+ 2 x[2]
+```
+
+We also developed the [MultiObjectiveAlgorithms.jl](https://github.com/jump-dev/MultiObjectiveAlgorithms.jl)
+(MOA) package, which implements a number of solution algorithms for multi-objective
+programs.
+
+The optimizer returns a set of points from the efficient frontier (exactly what
```
Comment (blegat) [inline _posts/2024-01-30-jump_year_in_review.md:166]: "exactly what and now" -> is that an english expression I don't know about ?

File: _posts/2024-01-30-jump_year_in_review.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+and now many depend on the solution algorithm), which you can access with the
```
Comment (blegat) [inline _posts/2024-01-30-jump_year_in_review.md:167]:
```suggestion
and how many depend on the solution algorithm), which you can access with the
```

File: _posts/2024-01-30-jump_year_in_review.md (hunk 1 above)
Comment (odow) [reply _posts/2024-01-30-jump_year_in_review.md:8]: See https://github.com/jump-dev/jump-dev.github.io/pull/118

**Comment by odow** [comment]
Now looks like:

image

**Comment by odow** [comment]
Can always revise if anyone has other suggestions

**Pull request merged**